examples/pip_repository_annotations/WORKSPACE

```diff
@@ -16,6 +16,14 @@ http_archive(
     ],
 )
 
+load("@rules_python//python:repositories.bzl", "python_register_toolchains")
+
+python_register_toolchains(
+    name = "python39",
```

Comment:
don't you want to use python 3.10 for at least one of the examples so you're proving the updated packages solve the problem you observed, and prevent regressions?

Reply:
That's a good call. I'll update one

Reply:
Done! 😄